packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -570,21 +576,31 @@ class ConfigScreenComponent extends React.Component<any, any> {
 				const onTextChange = (event: any) => {
 					updateSettingValue(key, event.target.value);
 				};
-
 				return (
 					<div key={key} style={rowStyle}>
 						<div style={labelStyle}>
 							<label>{md.label()}</label>
 						</div>
-						<input
-							type={inputType}
-							style={inputStyle}
-							value={this.state.settings[key]}
-							onChange={(event: any) => {
-								onTextChange(event);
-							}}
-							spellCheck={false}
-						/>
+						{
+							md.label() === 'Editor font family' || md.label() === 'Editor monospace font family' ?
```

Comment:
Rather than comparing `md.label()` with a hardcoded string, consider adding a new [setting `subType`](https://github.com/laurent22/joplin/blob/2488dd3806a1ee4e220b374eaeb3494f79adc1ee/packages/lib/models/Setting.ts#L56) to the font settings. This approach would be similar to how the file/directory chooser is currently handled.

Comparing the label with a hardcoded string could be problematic if the label for one of the font settings is updated at some point in the future or if the user changes Joplin to a non-English language.